draft-ietf-rtcweb-jsep.xml

```diff
@@ -3901,63 +3901,59 @@ v=0
 o=- 4962303333179871722 1 IN IP4 0.0.0.0
 s=-
 t=0 0
-a=group:BUNDLE a1 v1
 a=ice-options:trickle
+a=group:BUNDLE a1 v1
 m=audio 56500 UDP/TLS/RTP/SAVPF 96 0 8 97 98
 c=IN IP4 192.0.2.1
 a=mid:a1
-a=rtcp:56501 IN IP4 192.0.2.1
 a=msid:47017fee-b6c1-4162-929c-a25110252400
-       f83006c5-a0ff-4e0a-9ed9-d3e6747be7d9
 a=sendrecv
 a=rtpmap:96 opus/48000/2
 a=rtpmap:0 PCMU/8000
 a=rtpmap:8 PCMA/8000
 a=rtpmap:97 telephone-event/8000
 a=rtpmap:98 telephone-event/48000
 a=maxptime:120
-a=ice-ufrag:ETEn1v9DoTMB9J4r
+a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
+a=extmap:2 urn:ietf:params:rtp-hdrext:sdes:mid
+a=ice-ufrag:ETEn
 a=ice-pwd:OtSK0WpNtpUjkY4+86js7ZQl
 a=fingerprint:sha-256
-              19:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04
-             :BB:05:2F:70:9F:04:A9:0E:05:E9:26:33:E8:70:88:A2
+              19:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04:
+              BB:05:2F:70:9F:04:A9:0E:05:E9:26:33:E8:70:88:A2
 a=setup:actpass
+a=dtls-id:1
+a=rtcp:56501 IN IP4 192.0.2.1
 a=rtcp-mux
 a=rtcp-rsize
```

Comment:
For this PR I would like to avoid adding any new stuff to the SDP, just fixing the existing stuff. I will add the appropriate new stuff as part of #356.